StatisticOperations/StatVariance.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace StatisticOperations
+{
+    public class StatVariance
+    {
+        public static dynamic result;
+        public static dynamic Variance(dynamic arrayA)
+        {
+
+            dynamic theMean = StatMean.Mean(arrayA);
+            int arrayCount_1 = arrayA.Length - 1;
```

Comment:
arrayA.Length - 1 is for samples. We are doing calculations for populations. Please change this before merging